### DISCUSSION TITLE
Can i use tshock to change my game mode with mods?

### DISCUSSION THREAD

**Discussion by kryptex617** [Pryaxis/TShock#2169] [Q&A] [answered]
Can i use TShock to change my game mode to expert with mods like calamity downloaded on TModloader on steam?

**Comment by bartico6** [comment, marked as answer]
You cannot run tShock with tModLoader. Stuffing the world in, running /expert and saving might work, but in that case you might be better off using tEdit for that.

**Reply by Kojirremer** [reply]
As an addendum, you would have to be careful of which version of TEdit you are going to use, since latest build is for 1.4.1.1 worlds, and tModLoader is still in the 1.3.x range.
There is probably a mod that lets you change the world to expert anyway. Maybe HERO's, since IIRR it sort of emulated a chunk of TShock's features.

**Reply by kryptex617** [reply]
Thank you for the help the answer was indeed more mods lol I found one that let me change the difficulty
